climakitae/core/data_load.py

```diff
@@ -270,8 +272,10 @@ def area_subset_geometry(selections):
     """
     area_subset, cached_area = _override_area_selections(selections)
 
-    def set_subarea(boundary_dataset):
-        return boundary_dataset[boundary_dataset.index == shape_index].iloc[0].geometry
+    def set_subarea(
+        boundary_dataset: Boundaries, shape_indicies: list
+    ) -> gpd.GeoDataFrame:
```

Comment:
Yeah, I can change the import to `from geopandas import GeoDataFrame` instead, so it only imports the `GeoDataFrame` object instead of the whole library.